test/tortoise/connection/backoff_test.exs

```diff
@@ -8,18 +8,18 @@ defmodule Tortoise311.Connection.BackoffTest do
     min = 100
     max = 300
     backoff = Backoff.new(min_interval: 100, max_interval: 300)
-    assert {^min, backoff} = Backoff.next(backoff)
+    assert {value, backoff} = Backoff.next(backoff)
+    assert value >= min
     {_, backoff} = Backoff.next(backoff)
-    assert {^max, backoff} = Backoff.next(backoff)
-    # should roll back to min interval now
-    assert {^min, _} = Backoff.next(backoff)
+    {value, _} = Backoff.next(backoff)
+    assert value <= max
```

Comment:
```suggestion
    {value, backoff} = Backoff.next(backoff)
    assert value == min * 2
    assert {^max, _} = Backoff.next(backoff)
```
I couldn't make the suggestion change the first line. The intention is to check that the backoff doubles and then is capped at the max value.